convert-policy.py

```diff
@@ -97,11 +94,10 @@ def main():
         (title, policy_markdown) = markdown_title(policy_markdown)
         preprocessed_policy_markdown = preprocess_markdown(policy_markdown, link_mapping_pairs)
 
-        policy_html = markdown.markdown(preprocessed_policy_markdown, extensions=[PartialGithubFlavoredMarkdownExtension()])
+        policy_html = commonmark.commonmark(preprocessed_policy_markdown)
 
-        final_policy_html = template.replace("@POLICY_GOES_HERE@", policy_html)
-        final_policy_html = final_policy_html.replace("@TITLE_GOES_HERE@", title)
+        postprocessed_policy_html = postprocess_html(policy_html, template, title)
 
-        codecs.open("whatwg.org/" + link, "w", encoding="utf-8").write(final_policy_html)
+        codecs.open("whatwg.org/" + link, "w", encoding="utf-8").write(postprocessed_policy_html)
```

Comment:
I wonder why we're using this instead of just `open()`.

It looks like this might be a Python 2ism so we can just move to open (and remove the codecs import): https://stackoverflow.com/questions/5250744/difference-between-open-and-codecs-open-in-python

Reply:
Yeah I can do that.